### ISSUE TITLE
Define Baseline Training Configuration

### ISSUE THREAD

**Issue by giustacchini** [giustacchini/rl-assignment-2#15]
- Choose initial values for learning rate, discount factor, batch size, training duration, etc.
- Establish the baseline before experimenting.
